vtex-apps/condition-layout
# Add product-specification condition

#### What is the purpose of this pull request?

Add a new condition with a product specification filter.

#### What problem is this solving?

Add a new property where the condition meets the past product specification.

### How should this be manually tested?
- Enter this link: https://condition--tackoftheday.myvtex.com/elliena-eq-kids-the-edge-polo-470453-greygreystripe/p
- The "Size-Chart" tab-item will not appear because the product does not have this specification;
- Enter this link: https://condition--tackoftheday.myvtex.com/horseware-womens-softie-socks-triangle-print-cohfcr-8tri/p
- Now the "Size-Chart" tab-item will appear because the product has this "Size Chart:" specification.

### Screenshots or example usage
https://cl.ly/487487800767

Types of changes
